toolkit/javascripts/live-search.js

```diff
@@ -102,6 +103,14 @@ endpoint response (application/json):
     }
   };
 
+  LiveSearch.prototype.filterSelected = function(){
+    if(this.checked) {
+      var key = $(this).attr('name');
+      var value = $(this).attr('value');
+      GOVUK.GDM.analytics.virtualPageViews.trackFilterAnalytics(key,value);
```

Comment:
This feels like it will need some refactoring. The frontend-toolkit shouldn't have a dependency on a frontend-app. This piece of code is only implemented in the buyer-frontend and is otherwise a hidden bit of code you need to know to add when you want to have a LiveSearch feature.

Can we implement this `trackFilterAnalytics` in the frontend-toolkit?